go.mod

```diff
@@ -0,0 +1,98 @@
+module github.com/konveyor/openshift-migration-plugin
+
+go 1.14
+
+require (
+	github.com/BurntSushi/toml v0.3.1 // indirect
+	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5 // indirect
+	github.com/Microsoft/hcsshim v0.8.7 // indirect
+	github.com/VividCortex/ewma v1.1.1 // indirect
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cespare/xxhash/v2 v2.1.1 // indirect
+	github.com/containerd/cgroups v0.0.0-20191217135530-f26d9e0c03a9 // indirect
+	github.com/containerd/continuity v0.0.0-20191214063359-1097c8bae83b // indirect
+	github.com/containers/image/v5 v5.1.1-0.20200108174622-6dbed6e4847c
+	github.com/containers/libtrust v0.0.0-20190913040956-14b96171aa3b // indirect
+	github.com/containers/ocicrypt v0.0.0-20191112201931-142388cb70de // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/docker/distribution v2.7.1+incompatible // indirect
+	github.com/docker/docker v1.4.2-0.20180522102801-da99009bbb11 // indirect
+	github.com/docker/docker-credential-helpers v0.6.3 // indirect
+	github.com/docker/go-connections v0.4.0 // indirect
+	github.com/docker/go-metrics v0.0.1 // indirect
+	github.com/docker/go-units v0.4.0 // indirect
+	github.com/etcd-io/bbolt v1.3.3 // indirect
+	github.com/fatih/color v1.7.0 // indirect
+	github.com/fullsailor/pkcs7 v0.0.0-20190404230743-d7302db945fa // indirect
+	github.com/ghodss/yaml v1.0.0 // indirect
+	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/golang/groupcache v0.0.0-20191027212112-611e8accdfc9 // indirect
+	github.com/golang/protobuf v1.3.3-0.20191022195553-ed6926b37a63 // indirect
+	github.com/google/gofuzz v1.0.0 // indirect
+	github.com/googleapis/gnostic v0.3.1 // indirect
+	github.com/gorilla/mux v1.7.3 // indirect
+	github.com/hashicorp/go-hclog v0.10.0 // indirect
+	github.com/hashicorp/go-plugin v1.0.1-0.20190610192547-a1bc61569a26 // indirect
+	github.com/hashicorp/yamux v0.0.0-20190923154419-df201c70410d // indirect
+	github.com/imdario/mergo v0.3.8 // indirect
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/json-iterator/go v1.1.7 // indirect
+	github.com/klauspost/pgzip v1.2.1 // indirect
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/konveyor/openshift-velero-plugin v0.0.0-20200303232324-d6e74edcc40f
```

Comment:
This is the latest commit on master. Didn't realize this was how the mod file worked but makes sense. So when I do `go get github.com/konveyor/openshift-velero-plugin@master` it fills out go.mod with the latest commit hash.

Looks good! Didn't realize this was how it worked

Reply:
I have one question about this -- does this mean that the fact that openshift-velero-plugin tracks master is no longer recorded in the config (like it was with dep)? This means that the developer updating things needs to remember that on master, run `go get github.com/konveyor/openshift-velero-plugin@master`, but on release-1.1, run `go get github.com/konveyor/openshift-velero-plugin@release-1.1` (or whatever). Also, do we have the same issue with tracking velero -- or is the fact that on our fork, konveyor-dev is the branch we want already embedded in here?

Reply:
Yes you're right about not being able to track master dep. And the reason behind seems to be because of transient deps (If module A depends on B and C, and B depends on D@master, and C depends on D@dev, what should happen if I go get -u within A?)

On dev or master branch it's needed to manually on dev or master branch run `go get github.com/konveyor/openshift-velero-plugin@master`. 
But any other released branch will point to a released version.

Regarding the velero dep, I think that since github.com/vmware-tanzu/velero supports go modules the matter is just to find the right release or tagged version.